### PR TITLE
solves #200 and installation adjustments

### PR DESCRIPTION
**Pull request by mlodic**
softened the requirements + splitted test/prod requirements + removed duplicates + adjusted setup.py

### CODE REVIEW

**Review by SamuelHassine — APPROVED** [review]

**Comment by SamuelHassine** [comment]
Thanks a lot @mlodic for this work! We would like to merge your PR before our 5.0.0 release. Can you please solve the conflicts so we can move forward! Kind regards.

**Comment by mlodic** [comment]
done! I added the missing library from the upstream!

File: requirements.txt [hunk 1]
```diff
@@ -1,14 +1,10 @@
...
+requests~=2.26
+PyYAML~=5.4
+setuptools~=56.0
+datefinder~=0.7
+stix2~=2.1
+pika~=1.2
+sseclient~=0.0.27
+python-magic~=0.4; sys_platform == 'linux' or sys_platform == 'darwin'
+python-magic-bin~=0.4; sys_platform == 'win32'
+pytest-cases~=3.6
```
Comment (nor3th) [inline requirements.txt:10]: I know I am a bit too meticulous here, but could you please move the `pytest-cases` requirement to the test-requirements. 
Thank you very much!

Comment (mlodic) [reply requirements.txt]: you are right, just done it

Comment (mlodic) [reply requirements.txt]: feel free to squash the commits

Comment (nor3th) [reply requirements.txt]: Awesome, thanks!

**Review by nor3th — APPROVED** [review]
Looks good to me! Thank you for your help

**Comment by mlodic** [comment]
we are about to make a release in the next days of [IntelOwl](https://github.com/intelowlproject/IntelOwl) that leverages this library (10th of september last day). Could you please help us in merging and making a fix-release as soon as you can so we can avoid to introduce older dependencies?

Thank you very much in avdance

**Pull request merged**